src/Managers/MessageManager.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace OwowAgency\Gossip\Managers;
+
+use Illuminate\Support\Facades\Auth;
+use OwowAgency\Gossip\Support\Collection\MessageCollection;
+
+class MessageManager
+{
+    /**
+     * Create a new Eloquent Collection instance of messages.
+     *
+     * @param  array  $models
+     * @return \OwowAgency\Gossip\Support\Collection\MessageCollection
+     */
+    public static function createCollection(array $models): MessageCollection
```

Comment:
Still think this belongs in controller. This is just like going anal, going vag and than checking for condom.